frontend/src/query_builder/GuiQueryEditor.jsx

```diff
@@ -59,8 +61,8 @@ export default class GuiQueryEditor extends Component {
         }
     };
 
-    setQuery(dataset_query) {
-        this.props.setQueryFn(dataset_query);
+    setQuery(query) {
+        this.props.setQueryFn(query);
```

Comment:
Yeah, that happened mainly because for naming things in JSON we used underscores and because the JS code has a pretty close affiliation with the JSON objects going back and forth between the api it's often easiest to just keep the naming consistent.  So there are times when we are using the underscore naming instead of camelCase mainly just to avoid having to constantly translate back and forth.

In this case it would be fine as `datasetQuery` since it's just a function and not directly related to a JSON attribute.